ui/apps/platform/src/providers/FeatureFlagProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useMemo, type ReactNode } from 'react';
+
+import { type IsFeatureFlagEnabled, FeatureFlagsContext } from 'hooks/useFeatureFlags';
```

Comment:
Might this `type` have come from autocomplete/autoinsert?

What do you think are pro and con of separate `import type` statement?